viewer/add-heading-anchors.js

```diff
@@ -83,7 +83,7 @@ var addHeadingAnchors = {
 
   createSuccessTooltip: function (anchor) {
     $(anchor).tooltip({
-      title: "Link copied!",
+      title: "Link Copied!",
```

Comment:
This, I had forgotten to bring over, though it was sitting in a commit from November.  doh :/